v20.2/restore.md

```diff
@@ -88,19 +93,14 @@ When you restore a full cluster with an enterprise license, it will restore the
 
 #### Databases
 
-Restoring a database will create a new database and restore all of its tables and views.
-
-The created database will have the name of the database in the backup. The database cannot already exist in the target cluster.
-
-If [dropping](drop-database.html) or [renaming](rename-database.html) an existing database is not an option, you can use _table_ restore to restore all tables into the existing database:
+Restoring a database will create a new database and restore all of its tables and views. The created database will have the name of the database in the backup. The database cannot already exist in the target cluster.
```

Comment:
~Could the last sentence in this para re: "it can't already exist" be moved to the front?  I think that is what is most likely to make this go boom for users (didn't we see a forum post about this recently?)  I think making it bold would also be fine. :-}